### DISCUSSION TITLE
My fork:Chromebrew+(beta)

### DISCUSSION THREAD

**Discussion by YeesterPlus** [skycocker/chromebrew#9503] [Show and tell]
[upgrade/install](https://raw.githubusercontent.com/ChromebrewPlus/ChromebrewPlus/master/install.sh)
[repository](https://github.com/ChromebrewPlus/ChromebrewPlus)
to install/upgrade: run the "upgade/install" script

**Comment by satmandu** [comment]
Do you want to summarize your changes?

**Reply by YeesterPlus** [reply]
upgrade in install script,  removal of bad dep remove and replace with `Why am I still here, Why did I suffer`

**Reply by YeesterPlus** [reply]
and broken old tracker3 pkg (T ᷼ T)

**Reply by YeesterPlus** [reply]
and old packages

**Reply by YeesterPlus** [reply]
And google play chromebook opportunity

**Reply by ghost** [reply]
gotta show all comments!

**Comment by YeesterPlus** [comment]
letme ping everyone, @everyone

**Reply by YeesterPlus** [reply]
@supechicken @uberhacker @skycocker

**Comment by skycocker** [comment]
Blocked him, reported as an abuser to github, not much more I have time for now. Borderline is a horrible disorder.

**Reply by ghost** [reply]
please unmark, cuz I did
